docs/network-flow-visibility.md

```diff
@@ -634,3 +646,211 @@ With filters applied:
 
 <img src="https://downloads.antrea.io/static/03022021/flow-visualization-np-2.png" width="900" alt="Flow
 Visualization Network Policy Dashboard">
+
+### Grafana Flow Collector
+
+#### Purpose
+
+Antrea supports sending IPFIX flow records through the Flow Exporter and Flow Aggregator
+feature described above. The Grafana Flow Collector works as the visualization tool
+for flow records and flow-related information. We use ClickHouse as the data storage,
+which collects flow records data from Flow Aggregator and load the data to Grafana.
+This document provides the guidelines for deploying Grafana Flow Collector
+with support for Antrea-specific IPFIX fields in a Kubernetes cluster.
+
+#### About Grafana and ClickHouse
+
+[Grafana](https://grafana.com/grafana/) is an open-source platform for monitoring
+and observability. Grafana allows you to query, visualize, alert on and understand
+your metrics. [ClickHouse](https://clickhouse.com/) is an open-source, high performance
+columnar OLAP database management system for real-time analytics using SQL. We use
+ClickHouse as the data storage, and use Grafana as the data visualization and monitoring tool.
+
+#### Deployment Steps
+
```

Comment:
Should we encourage users to edit the `clickhouse-secret` `Secret` and use non-default credentials? I think that may be a good idea.

Reply:
Thanks, I added a [credential configuration](https://github.com/antrea-io/antrea/blob/793923f0b4ffa68190aebf60cfcd3ef2b1f72a04/docs/network-flow-visibility.md#credentials-configuration) section in the latest commit for review, where I also added the Grafana login credentials configuration.  There's one issue: flow-aggregator.yml also has a Secret with the same contents as `clickhouse-secret`. Their namespaces are different, so they cannot share the same Secret. Users need to make the changes at two different locations. 

I feel it can be addressed if we later decide to merge `flow-aggregator` and `flow-visibility` namespaces into one. But at the current stage, I'm not sure whether we want to add this customization support. Do you have any opinions on that?

Reply:
that sounds fine for now. I want us to support Helm as an installation method later one, and I believe it will make it easier to keep everything consistent.